server/config/keys.js

```diff
@@ -0,0 +1,10 @@
+//This file holds any configuration variables we may need
```

Comment:
Do not include this file in the repo. This has your api key. This file path should be in the git ignore.

Reply:
If deleting keys.js is not a correct move, leave this conversation unresolved.